<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Convert Customs Code to DCE Format</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TH SarabunPSK</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>แยกไฟล์</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ใส่จำนวนต่อไฟล์</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtBoxCountOfSplit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>381</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TH SarabunPSK</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>จัดการไฟล์ข้อมูล</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="btnDestinationSelect">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>TH SarabunPSK</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>เลือก</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSourceSelect">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>TH SarabunPSK</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>เลือก</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TH SarabunPSK</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>ระบบช่วยแปลงข้อมูลพิกัดศุลกากรเข้า DCE V.0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSourcePath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TH SarabunPSK</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TH SarabunPSK</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>โปรดเลือกสถานที่จัดเก็บ</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chbUseSplit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TH SarabunPSK</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ต้องการแยกไฟล์ด้วย</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDestinationPath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TH SarabunPSK</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TH SarabunPSK</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TH SarabunPSK</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>โปรดเลือกไฟล์ต้นฉบับ</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TH SarabunPSK</family>
      <pointsize>11</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
